com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -792,6 +792,18 @@ half3 VertexLighting(float3 positionWS, half3 normalWS)
     return vertexLightColor;
 }
 
+half4 GetShadowMask(InputData inputData)
```

Comment:
Remove this function. It's not used anywhere anymore. 
Suggestion for follow up PR: provide a GetShadowMask(float2 lightmapUV)

Reply:
Fixed